source/agora/test/Flash.d

```diff
@@ -585,21 +590,28 @@ private class FlashListener : TestFlashListenerAPI
         return utxos;
     }
 
+    public void setEstimatedTxFee(in Amount fee)
+    {
+        this.estimated_tx_fee = fee;
+    }
+
     public Amount getEstimatedTxFee ()
     {
-        return Amount(1);
+        return this.estimated_tx_fee;
     }
 }
 
-private TestConf flashTestConf ()
+private TestConf flashTestConf (bool noFee = true)
 {
     import agora.node.Config;
 
     TestConf conf;
     conf.consensus.quorum_threshold = 100;
-    // TODO: remove this line when fees are handled
-    conf.consensus.min_fee = Amount(0);
-    conf.node.min_fee_pct = 0;
+    if (noFee)
```

Comment:
There is no reason to have seperate tests for TXs with/without fees. It also leads to duplicated tests. I would say drop this commit from this PR too.

Reply:
Right, this `if` part is temporary. The goal for issue #2494  is to remove all this `if` part as the comment, `// TODO: remove this line when fees are handled` suggested. My current blocking problem about the issue is to fix the error [the test PR](https://github.com/bosagora/agora/pull/2679) makes. And, we need to fix the problem about `settle tx` also in other way [as you commented](https://github.com/bosagora/agora/pull/2678#discussion_r752095583).
Thank you for the review.